src/os.cc

```diff
@@ -107,6 +107,8 @@ int detail::utf16_to_utf8::convert(basic_string_view<wchar_t> s) {
 
 namespace detail {
 
+using mode_t = int;
```

Comment:
Shouldn't this be wrapped in `#ifdef _WIN32`?

Reply:
this detrail is already wrapped in _WIN32 on line 79